tests/security/mokutil_sign.pm

```diff
@@ -0,0 +1,175 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test mokutil signing function - Create necessary key pairs,
+#          sign the kernel, import cert and boot with signed kernel.
+# Maintainer: wnereiz <wnereiz@member.fsf.org>
+# Tags: poo#45701
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+use registration 'add_suseconnect_product';
+use power_action_utils "power_action";
+
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    if (is_sle('>12')) {
```

Comment:
that would trigger for each service pack of 12 as well I assume. I guess it should be '>=15'

Reply:
My mistake, modified!